README.md

```
@@ -1,78 +1,76 @@
 <div align="center">
-  <a href="https://wizardsardine.com/liana" target="_blank">
-    <img src="liana-ui/static/logos/liana-app-icon.svg" width="140px" />
+  <br><br>
+  <a href="https://coincube.io" target="_blank">
+    <img src="coincube-ui/static/logos/coincube-logo.svg" width="440px" />
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)